src/Messages/TSMessagesManager.m

```diff
@@ -490,7 +492,28 @@ - (void)handleIncomingEnvelope:(OWSSignalServiceProtosEnvelope *)incomingEnvelop
         }];
         if (ignoreMessage) {
             // FIXME: https://github.com/WhisperSystems/Signal-iOS/issues/1340
-            DDLogInfo(@"%@ Received message from group that I left or don't know about, ignoring", self.tag);
+            DDLogInfo(@"%@ Received message from group that I left or don't know about.", self.tag);
+
+            NSString *recipientId = incomingEnvelope.source;
+
+            __block TSThread *thread;
+            [[TSStorageManager sharedManager].dbConnection
+                readWriteWithBlock:^(YapDatabaseReadWriteTransaction *_Nonnull transaction) {
+                    thread = [TSContactThread getOrCreateThreadWithContactId:recipientId transaction:transaction];
+                }];
+
+            NSData *groupId = dataMessage.group.id;
+            OWSAssert(groupId);
+            OWSSyncGroupsRequestMessage *syncGroupsRequestMessage =
+                [[OWSSyncGroupsRequestMessage alloc] initWithThread:thread groupId:groupId];
+            [self.messageSender sendMessage:syncGroupsRequestMessage
+                success:^{
+                    DDLogInfo(@"%@ Successfully sent Request Group Info message.", self.tag);
+                }
+                failure:^(NSError *error) {
+                    DDLogError(@"%@ Failed to send Request Group Info message with error: %@", self.tag, error);
+                }];
+
```

Comment:
@michaelkirk We could go a step further and persist the message and re-process it if/when we get the solicited group update. It'd be a nice refinement but doesn't seem essential, and would introduce a certain amount of surface area for bugs.